flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/core/testutils/ManuallyTriggeredDirectExecutor.java

```diff
@@ -32,8 +33,17 @@
  */
 public class ManuallyTriggeredDirectExecutor implements Executor {
```

Comment:
What is the difference of this class and `ManuallyTriggeredScheduledExecutor`? Could they be unified?